### PR TITLE
2.0-preview branch

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -16,19 +16,33 @@
...
-    "polymer": "polymer/polymer#^1.0.0",
-    "iron-meta": "polymerelements/iron-meta#^1.0.0"
+    "polymer": "Polymer/polymer#^2.0.0-rc.1",
+    "iron-meta": "PolymerElements/iron-meta#2.0-preview"
   },
   "devDependencies": {
-    "paper-styles": "polymerelements/paper-styles#^1.0.2",
-    "iron-component-page": "polymerelements/iron-component-page#^1.0.0",
-    "iron-icon": "polymerelements/iron-icon#^1.0.0",
-    "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
```
Comment (notwaldorf) [inline bower.json]: Why isn't `promise-polyfill` needed here?

File: iron-iconset-svg.html [hunk 2]
```diff
@@ -225,8 +230,9 @@
           cssText += '-webkit-transform:scale(-1,1);transform:scale(-1,1);';
         }
 
-        svg.setAttribute('viewBox', viewBox);
-        svg.setAttribute('preserveAspectRatio', 'xMidYMid meet');
+        svg.setAttributeNS(null, 'viewBox', viewBox);
```
Comment (notwaldorf) [inline iron-iconset-svg.html]: Why is this change needed? Not questioning, just trying to understand since they look the same to me :)

Comment (bicknellr) [inline iron-iconset-svg.html]: Not needed at all: it was a workaround for https://github.com/webcomponents/custom-elements/issues/14 (fixed) that I totally forgot about.

Comment (bicknellr) [inline iron-iconset-svg.html]: reverted

File: bower.json (hunk 1 above)
Comment (bicknellr) [inline bower.json]: I don't know why I removed this, I'll add it back.

Comment (bicknellr) [inline bower.json]: added
